src/formatDate.js

```diff
@@ -43,14 +43,34 @@
  * ) // '18.02.1997'
  *
  * @param {string} date
- * @param {string[]} fromFormat
- * @param {string[]} toFormat
+ * @param {string[]} fromForm
```

Comment:
There is a typing error in the function parameters. It should be 'fromFormat' instead of 'fromForm'